backend/app/controllers/ProjectsController.scala

```diff
@@ -168,22 +168,42 @@ class ProjectsController @Inject() (
               _ <- validate(
                 project.active,
                 s"Cannot invite to an inactive project ${project.key}")
-              // create invitation
-              invitationId = InvitationId()
-              _ <- invitationRepository.upsert(
-                JoinProjectInvitation(
-                  id = invitationId,
-                  invitedEmail = request.body.email,
-                  createDate = DateTime.now(),
-                  createdBy = subject.userReference,
-                  expiration = DateTime.now().plusDays(7),
-                  sharedByOrganisationReference = userOrg.organisationReference,
-                  projectReference = project.getReference(),
-                  role = request.body.role,
-                  outcome = None
-                ))
+              // auto-assign user to project if user is already member of the organisation the project is part of
+              maybeExistingUser <- userRepository.findByEmail(
+                request.body.email)
```

Comment:
nitpick, reads a bit nicer
```suggestion
              maybeExistingUser <- userRepository.findByEmail(request.body.email)
```